diff --git a/week2-b/random_forest_classifier.pkl b/week2-b/random_forest_classifier.pkl
index 72e8896..0620915 100644
Binary files a/week2-b/random_forest_classifier.pkl and b/week2-b/random_forest_classifier.pkl differ
diff --git a/week2-b/wandb/latest-run b/week2-b/wandb/latest-run
index 98a7cab..e72869a 120000
--- a/week2-b/wandb/latest-run
+++ b/week2-b/wandb/latest-run
@@ -1 +1 @@
-run-20230604_104858-j3ea92k0
\ No newline at end of file
+run-20230604_113048-j4hpamew
\ No newline at end of file
