diff --git a/week2-b/random_forest_classifier.pkl b/week2-b/random_forest_classifier.pkl
index 72e8896..740e069 100644
Binary files a/week2-b/random_forest_classifier.pkl and b/week2-b/random_forest_classifier.pkl differ
diff --git a/week2-b/wandb/latest-run b/week2-b/wandb/latest-run
index 98a7cab..19da094 120000
--- a/week2-b/wandb/latest-run
+++ b/week2-b/wandb/latest-run
@@ -1 +1 @@
-run-20230604_104858-j3ea92k0
\ No newline at end of file
+run-20230604_112552-gu0flij8
\ No newline at end of file
